sdk/src/org.graalvm.polyglot/src/org/graalvm/polyglot/Context.java

```diff
@@ -1180,6 +1182,23 @@ public Builder allowCreateThread(boolean enabled) {
             return this;
         }
 
+        /**
```

Comment:
The corresponding TruffleContext API is missing.

Reply:
I am not really sure what's missing? Shall some methods be added to [TruffleContext](https://www.graalvm.org/truffle/javadoc/com/oracle/truffle/api/TruffleContext.html)?

Reply:
Christian says to add the method to https://www.graalvm.org/truffle/javadoc/com/oracle/truffle/api/TruffleContext.Builder.html